web/components/three.js/Warehouse.tsx

```diff
@@ -0,0 +1,20 @@
+import { Canvas } from "@react-three/fiber";
+import { OrbitControls, PerspectiveCamera } from "@react-three/drei";
+import Floor from "./Floor";
+import "./ThreeStyle.css"
+
+export default function Warehouse() {
+	return (
+		<div id="canvas-container">
```

Comment:
Non segue la sintassi di Tailwind.
canvas-container ha come unico attributo background: \#C09373
per creare colori personalizzati ti inoltro la guida ufficiale 
https://tailwindcss.com/docs/customizing-colors

in sostanza aggiungi un colore in tailwind.config.ts